CONTRIBUTING.md

```diff
@@ -0,0 +1,97 @@
+# HERMES development workflow
+
+## Preamble
+
+> **Branching is cheap!**
+>
+> Work in small packets, put up small pull requests!
+>
+> Aim for quick turnaround times!
+
+## Branching
+
+We loosely follow a mixture of [GitHubFlow](https://docs.github.com/en/get-started/quickstart/github-flow) and [GitFlow](https://nvie.com/posts/a-successful-git-branching-model/):
+
+`main`
+    - Stable branch
+    - Merges come only from `develop` or a hotfix branch (i.e., when something needs to be fixed in "production")
+
+`develop`
+    - Unstable development branch
+    - Last "stable" is tagged: tag before attempting to break something
+
+`feature/<describe-feature>` (including issue id when exists: `feature/62-improve-broken-thing`/`feature/42-add-new-thing`)
+    - Branch from last tag on `develop`
+
+`hotfix/<describe-hotfix>` (including issue id when exists: `hotfix/62-fix-broken-thing-in-release`)
+    - Branch from last tag on `main`
```

Comment:
Is branching from a tag not asking for merge conflicts?

Either the last tag on main is identical to main, or there are other hotfixes already on main, as merging a release branch should directly entail creating a new tag. Especially in a situation where we multiple hotfixes for the same release, we probably have better things to do than solving merge conflicts. After making a hotfix release, there should also be a new tag, i.e. the last tag is again identical to main.

Reply:
Agree, fixed in 6c5e869acfd2a8b8ffa605b6192b1ea506ec6afb.